### PR TITLE
Removed unused lambda captures

### PR DESCRIPTION
**Pull request by Dr15Jones**
This fixes warnings from clang.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

File: CommonTools/Utils/test/testFormulaEvaluator.cc [hunk 1]
```diff
@@ -781,7 +781,7 @@ testFormulaEvaluator::checkFormulaEvaluator() {
     std::vector<double> xValues = {.1, 1., 10., 100.};
 
 
-    auto func =[&v](double x) {return 100./3.*0.154227+2.36997;
+    auto func =[&](double x) {return 100./3.*0.154227+2.36997;
```
Comment (guitargeek) [inline CommonTools/Utils/test/testFormulaEvaluator.cc]: Hi, doesn't having simply an ampersand mean you capture all local variables? I guess clang also doesn't like that. To capture nothing you have to do `[]` I think.

Comment (Dr15Jones) [reply CommonTools/Utils/test/testFormulaEvaluator.cc]: clang doesn't actually care about `[&]` (i.e. capture needed variables by reference) it just cares if you explicitly capture a variable name but then never use it.

Comment (guitargeek) [reply CommonTools/Utils/test/testFormulaEvaluator.cc]: Ah ok, interesting! Thanks.

Comment (Dr15Jones) [reply CommonTools/Utils/test/testFormulaEvaluator.cc]: I changed to using `[]` anyway, since it is cleaner.

**Comment by Dr15Jones** [comment]
please test

**Comment by perrotta** [comment]
+1
- Technical
- Jenkins tests pass

**Comment by fabiocos** [comment]
+1

**Pull request merged**